packages/browser-agent-core/src/common/event-listener/event-listener-opts.js

```diff
@@ -14,4 +14,13 @@ try {
 
 export function eventListenerOpts(useCapture) {
   return supportsPassive ? {passive: true, capture: !!useCapture} : !!useCapture
+}
+
+/** Do not use this within the worker context. */
+export function windowAddEventListener(event, listener) {
+  window.addEventListener(event, listener, eventListenerOpts(false));
+}
+/** Do not use this within the worker context. */
+export function documentAddEventListener(event, listener) {
+  document.addEventListener(event, listener, eventListenerOpts(false));
```

Comment:
[question] Why are we creating functions that wrap `addEventListener`. These are pretty much just straight pass throughs and add bytes to the compiled code for seemingly no gain. Just wondering here.

Reply:
Yeah, so there's many if not all places where the `addEventListener` is called with that third `eventListenerOpts(useCapture)` arg that's meant to set the `options.passive` flag if avail. -- I have no why that's needed, but I was attempting to refactor all of the `addEventListener(_, _, eventListenersOpts(false))` -> `addEventListener(_, _)` instead.
(I haven't replaced all of them. If this has become irrelevant and just a relic, we can make a cleanup task for all of these instead.)